Library/Homebrew/cmd/tap.rb

```diff
@@ -18,6 +18,11 @@
 #:    if `--full` is passed, a full clone will be used. To convert a shallow copy
 #:    to a full copy, you can retap passing `--full` without first untapping.
 #:
+#:    By default, taps with repositories cloned from places other than GitHub
+#:    are skipped for performance reasons during the auto-update step that runs
+#:    before `brew install`, `brew upgrade` and `brew tap`. If `--force-auto-update` is
+#:    passed, this behavior will be disabled for the tap.
```

Comment:
Let's slim this down a bit. How about:

> By default, taps with non-GitHub repositories are not auto-updated (for performance reasons). If `--force-auto-update` is passed, this tap will be auto-updated even if it is not hosted on GitHub.

Reply:
> By default, taps with non-GitHub repositories are not auto-updated (for performance reasons).

Maybe also remove the double-negation here.

> By default, only taps hosted on GitHub are auto-updated (for performance reasons).

Reply:
@reitermarkus Agreed 👍